clients/web/src/templates/widgets/editCollectionWidget.pug

```diff
@@ -4,10 +4,10 @@
       .modal-header
         button.close(data-dismiss="modal", aria-hidden="true", type="button") &times;
         h4.modal-title
-          if (collection)
-            | Edit collection
-          else
+          if (create)
```

Comment:
The template, prior to this change, could be used without passing the `create` variable, but it now requires it, so if someone else was using it, it would break for them. It's probably dubious that anybody was using it, but since there's an easy equivalent non-breaking change, we should go with that.